src/popup/router/components/AddAccountCard.vue

```diff
@@ -1,20 +1,26 @@
 <template>
-  <div class="add-account-card">
+  <ButtonPlain
+    class="add-account-card"
+    @click="createAccount"
```

Comment:
Clicking on the add account card should not create new account immediately. It should open a separate modal. There's another task for this ready to be merged.